index.html

```diff
@@ -14,6 +14,75 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="container">
+    <nav>
+      <a href="#">Home</a>
+      <a href="#">About</a>
+      <a href="#">Portfolio</a>
+      <a href="#">Contact</a>
+    </nav>
+
+    <div class="top-container">
+      <section class="top-content">
+        <img src="Images/512.png">
+        <h1>Kevin Lopez - Fullstack Web Dev</h1>
+        <h4>I make websites</h4>
+        <hr>
+        <p>I'm a full time student at Lambda School</p>
+        <button>Contact me</button>
+      </section>
+    </div>
```

Comment:
Not so sure if you need both a top-container div and  a section container.  a section container might suffice.